code-challenges.test.js

```diff
@@ -1,49 +1,172 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
 const nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
+// // Expected output: [-8, 90]
 const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
+describe("minmumMaxium", () => {
+  it ("takes in an array of numbers and returns an array of the minimum and maxium numbers in that order", () => {
+       const nums1 = [3, 56, 90, -8, 0, 23, 6]
+       const nums2 = [109, 5, 9, 67, 8, 24]  
+       expect(minmunMaxium(nums1)).toEqual(expect.arrayMinmax([-8, 90]))
+       expect(minmunMaxium(nums2)).toEqual(expect.arrayMinmax([5, 109]))
+  })  
+ })
+//  FAIL  ./code-challenges.test.js
+//   minmumMaxium
+//     ✕ takes in an array of numbers and returns an array of the minimum and maxium numbers in that order
 
-// b) Create the function that makes the test pass.
+//   ● minmumMaxium › takes in an array of numbers and returns an array of the minimum and maxium numbers in that order
 
+//     ReferenceError: minmunMaxium is not defined
+
+// b) Create the function that makes the test pass.
+//Pseudo Code
+//Create a function 
+//The function will be called "minimumMaxium"
+//the Array will returned the smallest & biggest number order.  
+//Iterate the values in the array 
+//use the spread operator 
+//create variables for Math.min and Math.max
+//output: [-8,90]
+//output: [5, 109]
+// minimumMaximum= (array) 
+// var new_array = []
+// for (var i = 0; i < array.length; i++){
+// if (array[i] === Math.max.apply(Math, array)){
+//  new_array.unshift(array[i])
+//  else if (array[i] === Math.min.apply(Math, array)) 
+
+  const minimumMaximum =(arrary) => {
+    let minimum = Math.min(...arraryarray) 
+    let maximum = Math.max(...array)
+    let result = ([minimum, maximum])
+    return result
+  }
+//   PASS  ./code-challenges.test.js
+//   minmumMaxium
+//     ✓ takes in an array of numbers and returns an array of the minimum and maxium numbers in that order (1 ms)
+// Test Suites: 1 passed, 1 total
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const testArray1 = [3, 7, 10, 5, 4, 3, 3]
-const testArray2 = [7, 8, 2, 3, 1, 5, 4]
-// Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
-
-
+// const testArray1 = [3, 7, 10, 5, 4, 3, 3]
+// const testArray2 = [7, 8, 2, 3, 1, 5, 4]
+// //Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+
+describe("twoarrays", () => {
+    it ("takes in two arrays as arguments and returns one array with no duplicate values", () => {
+        const testArray1 = [3, 7, 10, 5, 4, 3, 3]
+        const testArray2 = [7, 8, 2, 3, 1, 5, 4]
+//      expect(twoarrays(testArrary1,testArray2).toEqual(expect.arrayoutput([3, 7, 10, 5, 4, 3, 3])[7, 8, 2, 3, 1, 5, 4]))
+ })
+ }) 
```

Comment:
Expect statement should not be commented out.  Also, keep in mind what you are trying to compare here.  The first part of your expect statement is invoking the function and passing in the arguments of your test variables.  The second part of the expect statement after the .toEqual is checking if the result from your function strictly equals the **_expected output_**.